src/Lecture1.hs

```diff
@@ -121,6 +135,11 @@ Explanation: the list [1 .. 9] contains 9 elements: [1, 2, 3, 4, 5, 6, 7, 8, 9]
 The given number 3 is greater than 2 elements (1 and 2)
 and lower than 6 elements (4, 5, 6, 7, 8 and 9).
 
-🕯 HINT: Use recursion to implement this function.
+HINT: Use recursion to implement this function.
 -}
-lowerAndGreater n list = error "TODO"
+
+lowerAndGreater :: Int -> [Int] -> String
+lowerAndGreater n list = show n ++ " is greater than " ++ show lowerCount ++ " elements and lower than " ++ show higherCount ++ " elements"
+                          where countLowerHigher = foldl (\[l,h] x -> if x < n then [l + 1, h] else if x > n then [l, h + 1] else [l, h]) [0, 0]
```

Comment:
Great solution using `fold`! 🔝 🏆 

You can replace lists with tuples here to avoid having non-exhaustive pattern matching warning ⚠️ 
Generally, the specific `foldl` is not recommended because it's slow. But I'll cover folds in lecture 3 and this is already great 👍🏻 

```suggestion
                          where countLowerHigher = foldl (\(l,h) x -> if x < n then (l + 1, h) else if x > n then (l, h + 1) else (l, h)) (0, 0)
```